sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/csv/CSVSuite.scala

```diff
@@ -2927,17 +2974,17 @@ abstract class CSVSuite
       }
 
       check(
-        "legacy",
+        "corrected",
```

Comment:
to reduce the diff, can we still test `legacy` first?

Reply:
Done